### PR TITLE
NAS-102245 / 12.0 / Add graphite separate instances field to reporting settings

### PR DESCRIPTION
**Pull request by erinix**
NAS-102245

### CODE REVIEW

File: src/app/helptext/system/reporting.ts [hunk 1]
```diff
@@ -30,7 +30,13 @@ reset_button: T('Reset to Defaults'),
 dialog: {
     title: T('Change Settings and Clear Report History?'),
     message: T('Report history is cleared when <i>CPU reporting</i>, <i>Graph Age</i>, \
- or <i>Graph Points</i> are changed.'),
+<i>Graphite Separate Instances</i> or <i>Graph Points</i> are changed.'),
```
Comment (a-stjohn) [inline src/app/helptext/system/reporting.ts]:
```suggestion
<i>Graphite Separate Instances</i>, or <i>Graph Points</i> are changed.'),
```

File: src/app/helptext/system/reporting.ts [hunk 2]
```diff
@@ -30,7 +30,13 @@ reset_button: T('Reset to Defaults'),
...
     action: T('Continue')
-}
+},
+graphite_separateinstances_placeholder: T('Graphite Separate Instances'),
+graphite_separateinstances_tooltip:T("If checked, when sending to Graphite,\
```
Comment (Mrt134) [inline src/app/helptext/system/reporting.ts]:
```suggestion
graphite_separateinstances_tooltip:T("Enabling sends the \
 <i>plugin instance</i> and <i>type instance</i> to Graphite as separate \
 path components: <i>host.cpu.0.cpu.idle</i>.<br><br> Disabling sends \
 the <i>plugin</i> and <i>plugin instance</i> as one path component and \
 <i>type</i> and <i>type instance</i> as another component: \
 <i>host.cpu-0.cpu-idle</i>.")
```
Suggestion unifies tense, removes asides and pauses, and adds an empty space between set/unset descriptions to improve readability.

File: src/app/helptext/system/reporting.ts [hunk 3]
```diff
@@ -30,7 +30,13 @@ reset_button: T('Reset to Defaults'),
...
+ the plugin instance and type instance will be in their own path component,\
+ for example host.cpu.0.cpu.idle. If unchecked (the default), the plugin and\
+ plugin instance  (and likewise the type and type instance) are put into one component,\
+ for example host.cpu-0.cpu-idle.")
```
Comment (a-stjohn) [inline src/app/helptext/system/reporting.ts]: Suggested rewrite:
```
The plugin, plugin instance and type, type instance are combined into one component. \
For example, <i>host.cpu-0.cpu-idle</i>. When set, the plugin and type instance will \
be in their own path component. For example, <i>host.cpu.0.cpu.idle</i>.
```

**Comment by a-stjohn** [comment]
> 
> 
> Slight re-work suggestion.

Ignore my review. It appears Tim and I were reviewing it at the same time and he beat me to it.

**Pull request merged**